docs/index.md

```diff
@@ -6,9 +6,11 @@ page: true
 
 <script setup>
 
-if (typeof window !== 'undefined') {
-  const preferredLang = localStorage.getItem('preferred_lang') || 'en-US'
-  window.location.pathname = `/${preferredLang}/`
-}
+  if (typeof window !== 'undefined') {
+    const preferredLang = localStorage.getItem('preferred_lang') || navigator.language || ''
+    const supportedLang = ['en-US', 'es-ES', 'fr-FR', 'ja-JP', 'zh-CN']
```

Comment:
I think languages should not be hardcode here, useTranslation hook should provide all supported languages

Reply:
Yes, that's a good idea.